website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -81,7 +81,8 @@ The Appsmith server should be up and running soon and can be accessed at [http:/
 
 ### Updating Appsmith
 :::caution
-   It's recommended to backup the Appsmith instance before performing an update. For more information, see [How to create a backup](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance).
+   * It's recommended to backup the Appsmith instance before performing an update. For more information, see [How to create a backup](https://docs.appsmith.com/getting-started/setup/instance-management/appsmithctl#backup-appsmith-instance).
+* When you are manually updating the Appsmith instance, ensure that you [upgrade to the checkpoint](/getting-started/setup/instance-management#checkpoint-upgrade) version `v1.9.2` released between your version and the latest version.
```

Comment:
Let's specify saying, if and only if your current `Appsmith version < v1.9.2` then follow the instructions to upgrade to checkpoint.